### PR TITLE
MAKE-1229: Add new management methods to cli

### PR DESCRIPTION
**Pull request by julianedwards**

### CODE REVIEW

File: cli/queue_management.go [hunk 1]
```diff
@@ -213,3 +215,54 @@ func managementReportRecentErrors(opts *ServiceOptions) cli.Command {
 		},
 	}
 }
+
+func managementCompleteJob(opts *ServiceOptions) cli.Command {
+	return cli.Command{
+		Name: "complete_job",
```
Comment (tychoish) [inline cli/queue_management.go:221]: I would use hyphens rather than underscores

File: cli/queue_management.go [hunk 2]
```diff
@@ -213,3 +215,54 @@ func managementReportRecentErrors(opts *ServiceOptions) cli.Command {
...
+		Flags: opts.managementReportFlags(
+			cli.StringFlag{
+				Name:  "name",
```
Comment (tychoish) [inline cli/queue_management.go:224]: should this be a constant.

File: cli/queue_management.go [hunk 3]
```diff
@@ -213,3 +215,54 @@ func managementReportRecentErrors(opts *ServiceOptions) cli.Command {
...
+				return nil
+			})
+		},
+	}
+
+}
+
+func managementCompleteJobByType(opts *ServiceOptions) cli.Command {
+	return cli.Command{
+		Name: "complete_job_by_type",
```
Comment (tychoish) [inline cli/queue_management.go:247]: rename

File: cli/queue_management.go [hunk 4]
```diff
@@ -213,3 +215,54 @@ func managementReportRecentErrors(opts *ServiceOptions) cli.Command {
...
+		Flags: opts.managementReportFlags(
+			cli.StringFlag{
+				Name:  "type",
```
Comment (tychoish) [inline cli/queue_management.go:250]: constant.

File: rest/client_management.go [hunk 5]
```diff
@@ -130,6 +130,34 @@ func (c *ManagementClient) RecentJobErrors(ctx context.Context, jobType string,
...
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return errors.Wrap(err, "error processing request")
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		var msg string
+		data, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			msg = fmt.Sprintf("problem reading response body: '%s'", err.Error())
```
Comment (tychoish) [inline rest/client_management.go:151]: this shld just be error wrapping

Comment (julianedwards) [reply rest/client_management.go:151]: it is on line 155 below

**Pull request merged**